src/apps/util/config.clj

```diff
@@ -102,10 +102,10 @@
   [props config-valid configs]
   "apps.tools.private.time-limit-seconds" (* 24 60 60)) ;; 24 hours
 
-(cc/defprop-optint private-tool-cpu-shares
```

Comment:
Right, these weren't added to the config repos since they're intended as service defaults (and if needed the service can use a different value by updating the live config file).

I'm not sure we need them in the config files yet (we don't need different values in different environments).